content/beginner/200_secrets/_index.md

```diff
@@ -0,0 +1,16 @@
+---
+title: "Managing Secrets Deployment"
+chapter: true
+weight: 200
+tags:
+  - beginner
+  - CON206
+---
+
+# Managing Secrets Deployment
```

Comment:
Given the scope of the chapter, I would like the title to be more crisp in relation to what problem the chapter is highlighting and solving. 'Secure Secret using BitNami SealedSecrets' or similar. This clearly informs the reader on value they'd get from this exercise.

Reply:
How about just "Secure Secrets using SealedSecrets" and leave Bitnami out of the title?

Reply:
Actually, I made it "Securing Secrets using SealedSecrets"

Reply:
LGTM